### ISSUE TITLE
Coqdoc's HTML output breaks UTF-8 character by splitting up its encoding.

### ISSUE THREAD

**Issue by Zimmi48** [Zimmi48/bugzilla-test#2275]
Note: the issue was created automatically with bugzilla2github tool

# Bugzilla Bug ID: 2275
Date: 2010-03-21 17:20:00 +0100
From: Eelis van der Weegen &lt;<coq-bugs@contacts.eelis.net>&gt;
To:   
CC:   @_herbelin

Last updated: 2010-03-30 21:20:25 +0200

## Bugzilla Comment ID: 3279
Date: 2010-03-21 17:20:00 +0100
From: Eelis van der Weegen &lt;<coq-bugs@contacts.eelis.net>&gt;

Consider a file t.v with the following UTF-8 encoded contents:

  Notation "⟶" := 2.
  Check ⟶.

coqc has no problem with this file (it prints "2: nat" as expected).

However, if I run 

  coqdoc -utf8 t.v

Then the "⟶" symbol in the Check line is broken. Looking at the HTML source with hexdump, I find the following in the Check line:

  <span class="id" type="var">A</span>BC

where A, B, and C are the three bytes (0xE2, 0x9F, 0xB6) that make up the UTF-8 encoding of the "⟶" symbol.

In other words, Coqdoc has broken the "⟶" symbol by splitting up its UTF-8 encoding.

I'm using Coq trunk 12869.

## Bugzilla Comment ID: 3314
Date: 2010-03-30 21:20:25 +0200
From: @_herbelin

Fixed in revision 12895 (v8.3 and trunk). [This was due to an interference with obsolete support for iso-8859-1 in Coq identifiers]
